src/topology/algebra/continuous_functions.lean

```diff
@@ -304,6 +364,9 @@ def continuous_map.C : R →+* C(α, A) :=
   map_zero' := by ext x; exact (algebra_map R A).map_zero,
   map_add'  := λ c₁ c₂, by ext x; exact (algebra_map R A).map_add _ _ }
 
+@[simp] lemma continuous_map.C_apply (r : R) (a : α) : continuous_map.C r a = algebra_map R A r :=
+rfl
```

Comment:
What is the reason to have it this way and not the other way around? (I mean algebra_map = continuous_map.C)

Reply:
I was thinking that we already had a zillion lemmas about algebra_map. In the `stone_weierstrass2` branch, I needed this direction in order to have a proof finish by `simp`, so I took that as justification enough. :-)

Reply:
Ok that makes sense!